lessons/Data-Science/box-plots-interpret/langs/en-us/index.adoc

```diff
@@ -0,0 +1,384 @@
+- In the bottom section of the poster, sketch the *box plot* that would result in the distribution to match the histogram and scenario.
+}
+
+@teacher{
+Spend some time debriefing with students, using the prompts below. Responses will vary for each group of students.
+}
+
+@QandA{
+@Q{What strategies did you use to make sure that your histogram and box plots matched?}
+
+@Q{Did all of the box plots your classmates developed correctly represent the given scenario?}
+
+@Q{What information does a box plot convey that a histogram doesn't?}
+
+@A{What information does a histogram convey that a box plot doesn't?}
+
+}
+
+
+@lesson-instruction{
+- Use what you've learned about how box plots and histograms display data differently to complete @printable-exercise{matching-boxplots-to-histograms.adoc}.
+
+- @opt{Complete @opt-printable-exercise{matching-boxplots-to-histograms-2.adoc} and/or the Matching Box Plots to Histograms slide of @opt-starter-file{boxplots}}
+}
+
+@clear
+@slidebreak
+
+@lesson-instruction{
+Now that you have the skills to interpret box plots, complete @printable-exercise{data-cycle-box-plots-animals.adoc}.
+}
+
+@pd-slide{
+*Matching Debrief: Think about shape, don't rely on numbers!*
+
+There are teachers and students who complain about this page. They say that this page is unfair because we didn't label the axes. The stats teachers are always shocked at their colleagues. They say, "It doesn't matter! We're talking about shape. Skew right is skew right, no matter where it falls on the x axis.""
+
+So many kids have sort of a statistical lazy eye. How do you fix that? You wear a patch over the strong eye. By removing the numbers, it forces people to think about the shape. If the numbers are there, you get high achieving algebra students who never learn shape because they keep using the numbers.
+}
+
+
+
+=== Synthesize
+
+@QandA{
+@Q{Match the box plots below with the corresponding histograms. How did you make your matches?}
+@A{Box plot A represents the same data as histogram C. Box plot B represents the same data as histogram D. Students may suspect that the short whiskers of histogram B match with the short bars on the edges of histogram C. Because box plots represent density and not frequency, this reasoning is incorrect. }
+
+++++
+<style>
+/* Add letters to the top left corner, relative to each table cell */
+.lettering td { position: relative; }
+.lettering .paragraph:first-child p { position: absolute; top: 0; font-weight: bold; }
+</style>
+++++
+
+[.lettering, cols="^.^1a,^.^1a", header="none", stripes="none"]
+|===
+| A
+
+@image{images/launch-boxplot-a.png, 125}
+| B
+
+@image{images/launch-boxplot-b.png, 125}
+| C
+
+@image{images/launch-histo-c.png, 125}
+| D
+
+@image{images/launch-histo-d.png, 125}
+
+|===
+
+
+}
+
+@teacher{The matching activities in the Launch and Synthesize sections of this lesson were drawn from research conducted by @citation{lem-et-al-2011, "Lev et al (2011)"}.
```

Comment:
This box felt a little jumbled to me and like it left people hanging. I've pushed a change that fixes reference to dot plots that must've been intended to say box plots, reorder the ideas, and links to the dot plots to histograms lesson. Flagging, because I know these boxes are important to you.

Reply:
Thanks! Agree, it was a little sloppy.